x-pack/plugins/security_solution/public/flyout/document_details/right/context.tsx

```diff
@@ -8,8 +8,8 @@
 import type { BrowserFields, TimelineEventsDetailsItem } from '@kbn/timelines-plugin/common';
 import React, { createContext, memo, useContext, useMemo } from 'react';
 import type { EcsSecurityExtension as Ecs } from '@kbn/securitysolution-ecs';
-import { TableId } from '@kbn/securitysolution-data-table';
 
+import { TableId } from '@kbn/securitysolution-data-table';
```

Comment:
this change isn't necessary, let's remove it

Reply:
`TableId` is needed to check if the page is a rule preview